internal/tools/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/frapposelli/wwhrd v0.4.0
-	github.com/golangci/golangci-lint v1.53.3
+	github.com/golangci/golangci-lint v1.54.2
```

Comment:
We use the same version on the Collector https://github.com/open-telemetry/opentelemetry-collector/blob/064df806f1caea2d95aaa407ff792e806f60896b/internal/tools/go.mod#L9 so this should be fine